lib/rbs/inline/ast/members.rb

```diff
@@ -484,6 +484,57 @@ def attribute_type #: Types::t
           end
         end
 
+        class RubyIvar < RubyBase
+          # @rbs! type t = Prism::ClassVariableWriteNode | Prism::ClassVariableOrWriteNode
+          #              | Prism::InstanceVariableWriteNode | Prism::InstanceVariableOrWriteNode
```

Comment:
It would be nice to give a better name for such type alias. Please let me know if you have opinions.